pynance/data/retrieve.py

```diff
@@ -10,24 +10,27 @@
 Wraps `Pandas Remote Data Access 
 <http://pandas.pydata.org/pandas-docs/stable/remote_data.html>`_.
 """
-
+from datetime import datetime
 from ftplib import FTP
 from functools import partial
 import io
 
 import pandas_datareader.data as web
 
-def get(equity, start, end):
+def get(equity, *args, **kwargs):
     """ 
     Get DataFrame for an individual equity from Yahoo!  
-    
+
+    .. changed:: 0.5.0 - changed to variadic parameters
```

Comment:
The variadic parameters shouldn't be exposed to the casual user. More useful would be something like 'fill in missing start and end dates with defaults'.